packages/suite-desktop/src-electron/app.ts

```diff
@@ -9,6 +9,7 @@ import { MIN_HEIGHT, MIN_WIDTH } from '@desktop-electron/libs/screen';
 import Logger, { LogLevel, defaultOptions as loggerDefaults } from '@desktop-electron/libs/logger';
 import { buildInfo, computerInfo } from '@desktop-electron/libs/info';
 import modules from '@desktop-electron/libs/modules';
+import { isDev as isDevEnvironment } from '@suite-utils/build';
```

Comment:
i was confused because on electron side we use `import isDev from 'electron-is-dev';` in multiple places

should we unify that and use only `@suite-utils/build` or `packages/suite-build/utils/env` everywhere?

Reply:
`electron-is-dev` is using env `ELECTRON_IS_DEV` which has to be set to `1` (`dev` script in `suite-desktop/package.json` )

I will discuss it with @matejkriz if it is good idea to have both `ELECTRON_IS_DEV` and `NODE_ENV`. It sounds confusing and it can cause errors in the future.

Reply:
Not sure about unifying `@suite-utils/build` and `packages/suite-build/utils/env` but I believe we don't need `electron-is-dev` and we could remove it completely. The only difference from using NODE_ENV directly is, that electron-is-dev has a fallback to [`!electron.app.isPackaged`](https://github.com/sindresorhus/electron-is-dev/blob/main/index.js#L11)

Other thing is, that I don't understand why we have `isDev()` from @suite-utils/build defined as a function, because then the code is not stripped from production.

Reply:
9c50035fe50776881857c7645f6adca1f0c26323